phoenix-core/src/it/java/org/apache/phoenix/end2end/MaxLookbackIT.java

```diff
@@ -213,62 +241,49 @@ public void testRecentMaxVersionsNotCompactedAway() throws Exception {
         String thirdValue = "ghi";
         try (Connection conn = DriverManager.getConnection(getUrl())) {
             String dataTableName = generateUniqueName();
-            String indexStem = generateUniqueName();
-            createTableAndIndexes(conn, dataTableName, indexStem, versions);
-            long afterInsertSCN = org.apache.phoenix.util.EnvironmentEdgeManager.currentTimeMillis();
+            createTable(dataTableName);
+            //increment by 10 min to make sure we don't "look back" past table creation
+            injectEdge.incValue(WAIT_AFTER_TABLE_CREATION);
+            populateTable(dataTableName);
+            injectEdge.incValue(1); //increment by 1 so we can see our write
+            long afterInsertSCN = EnvironmentEdgeManager.currentTimeMillis();
             //make sure table and index metadata is set up right for versions
             TableName dataTable = TableName.valueOf(dataTableName);
             assertTableHasVersions(conn, dataTable, versions);
-            String fullIndexName = indexStem + "1";
-            TableName indexTable = TableName.valueOf(fullIndexName);
```

Comment:
What was the reason for eliminating index tables from the tests? Was not this JIRA originally about index tables?

Reply:
It was originally about index tables, but I eventually realized that this needed to apply to all tables in order for SCN to work properly, so there's nothing "index specific" about the functionality anymore.

If the consensus is that we should have specific index cases in here to make sure nothing in the indexing coprocs break the general behavior, I'll put some (back) in.

Reply:
I suggest including indexes back and applying the same operations/checks to the index tables along with their data tables.

Reply:
All right, will do.